runtime/lib/app-runtime.js

```diff
@@ -77,7 +55,7 @@ AppRuntime.prototype.init = function init () {
   this.componentsInvoke('init')
   /** set turen to not muted */
   this.component.turen.toggleMute(false)
-  this.component.turen.toggleWakeUpEngine(true)
+  //this.component.turen.toggleWakeUpEngine(true)
```

Comment:
Why removing this line?

Reply:
Just comment it for developing, as this line always result in error of undefined function.